dbt/adapters/dremio/api/rest/url_builder.py

```diff
@@ -31,92 +34,90 @@ class UrlBuilder:
     CLOUD_CATALOG_ENDPOINT = CLOUD_PROJECT_ENDPOINT + "/{}/catalog"
 
     @classmethod
-    def login_url(cls, base_url):
-        return base_url + UrlBuilder.SOFTWARE_LOGIN_ENDPOINT
+    def login_url(cls, parameters: Parameters):
+        return parameters.base_url + UrlBuilder.SOFTWARE_LOGIN_ENDPOINT
 
     @classmethod
-    def sql_url(cls, base_url, is_cloud=False, cloud_project_id=None):
-        if is_cloud:
-            return base_url + UrlBuilder.CLOUD_SQL_ENDPOINT.format(cloud_project_id)
-        return base_url + UrlBuilder.SOFTWARE_SQL_ENDPOINT
+    def sql_url(cls, parameters: Parameters):
+        if type(parameters) is CloudParameters:
+            return parameters.base_url + UrlBuilder.CLOUD_SQL_ENDPOINT.format(
+                parameters.cloud_project_id
+            )
+        return parameters.base_url + UrlBuilder.SOFTWARE_SQL_ENDPOINT
 
     @classmethod
-    def job_status_url(cls, base_url, job_id, is_cloud=False, cloud_project_id=None):
-        if is_cloud:
+    def job_status_url(cls, parameters: Parameters, job_id):
+        if type(parameters) is CloudParameters:
             return (
-                base_url
-                + UrlBuilder.CLOUD_JOB_ENDPOINT.format(cloud_project_id)
+                parameters.base_url
+                + UrlBuilder.CLOUD_JOB_ENDPOINT.format(parameters.cloud_project_id)
                 + "/"
                 + job_id
             )
-        return base_url + UrlBuilder.SOFTWARE_JOB_ENDPOINT + "/" + job_id
+        return parameters.base_url + UrlBuilder.SOFTWARE_JOB_ENDPOINT + "/" + job_id
 
     @classmethod
-    def job_cancel_url(cls, base_url, job_id, is_cloud=False, offset=0, limit=100):
-        url_path = None
-        if is_cloud:
-            url_path = base_url + UrlBuilder.CLOUD_JOB_ENDPOINT
+    def job_cancel_url(cls, parameters: Parameters, job_id, offset=0, limit=100):
```

Comment:
Remove offset and limit as they are not appropriate for this call (and are never used).
(I understand these were pre-existing but we should get rid of them.)